server/odc-migrate/src/main/java/com/oceanbase/odc/migrate/jdbc/common/V4324Oauth2UseStateParamDefaultFalse.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.migrate.jdbc.common;
+
+import java.util.List;
+
+import javax.sql.DataSource;
+
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import com.oceanbase.odc.common.json.JsonUtils;
+import com.oceanbase.odc.core.migrate.JdbcMigratable;
+import com.oceanbase.odc.core.migrate.Migratable;
+import com.oceanbase.odc.metadb.integration.IntegrationEntity;
+import com.oceanbase.odc.service.integration.model.Oauth2Parameter;
+import com.oceanbase.odc.service.integration.model.SSOIntegrationConfig;
+
+@Migratable(version = "4.3.2.4",
```

Comment:
you may use 4.3.2.5, 4.3.2.4 is occupied

Reply:
done!